tests/system/python/packages/test_rule_data_availability.py

```diff
@@ -263,6 +263,7 @@ def test_data_availability_north(self, check_eds_installed, reset_fledge, start_
         get_url = "/fledge/audit?source=NTFSN"
         resp1 = utils.get_request(fledge_url, get_url)
         
+        time.sleep(wait_time)
```

Comment:
We should look for a viable solution NOT a hardcoded wait time interim solution. 
We may add poll mechanism to receive the notification and retry based check in expected time